domains/_github-pages-challenge-smirkhat.sh.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SmirkHat",
```

```suggestion
        "username": "danielsebesta",
```
username is your github username